ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/tracker/CompletionResponse.scala

```diff
@@ -117,20 +117,30 @@ object CompletionResponse {
   }
 
   private def buildException(metadata: Map[String, String], status: StatusJavaProto.Builder) = {
-    val newDetails = status.getDetailsList.asScala.map { detail =>
```

Comment:
There was a bug as well, resulting in ignoring the metadata if the details were empty.